terra/devnet/config/genesis.json

```diff
@@ -431,7 +429,7 @@
     },
     "distribution": {
       "params": {
-        "community_tax": "0.000000000000000000",
+        "community_tax": "0.020000000000000000",
```

Comment:
The community tax is 0.5% now
https://station.terra.money/proposal/columbus-5/11515

Reply:
Thanks @eliiik. Personally I think the exact tax rate we test against doesn't matter so much, what's important is that we have a non-0 value here to ensure that the tax handling logic works as intended. The actual number will likely keep changing anyway.